safedelete/tests/test_soft_delete.py

```diff
@@ -153,6 +167,18 @@ def test_update_or_create_with_unique_field(self):
         self.assertEqual(obj.name, 'unique-test')
         self.assertEqual(created, False)
 
+    def test_update_or_create_with_unique_constraint(self):
+        self.assertFalse(UniqueConstraintSoftDeleteModel.has_unique_fields())
```

Comment:
This will eventually be assertTrue once my fix is implemented, but for now, this returns False